shared/build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 plugins {
     kotlin("multiplatform")
     id("com.android.library")
-    id("co.touchlab.faktory.kmmbridge") version "0.2.2"
+    id("co.touchlab.faktory.kmmbridge") version "0.3.1"
```

Comment:
0.3.2 now

Reply:
That actually wasnt working for me in this repo. On 0.3.1 we get this warning on sync
```
warn: From https://github.com/touchlab/KMMBridgeSampleKotlin
warn:  ! [rejected]        0.6.0      -> 0.6.0  (would clobber existing tag)
warn:  ! [rejected]        0.6.1      -> 0.6.1  (would clobber existing tag)
warn:  ! [rejected]        0.6.2      -> 0.6.2  (would clobber existing tag)
```

But thats been bumped up to an error in 0.3.2
```
error: From https://github.com/touchlab/KMMBridgeSampleKotlin
error:  ! [rejected]        0.6.0      -> 0.6.0  (would clobber existing tag)
error:  ! [rejected]        0.6.1      -> 0.6.1  (would clobber existing tag)
error:  ! [rejected]        0.6.2      -> 0.6.2  (would clobber existing tag)

```